aviary/interface/methods_for_level2.py

```diff
@@ -231,6 +231,9 @@ def __init__(self, analysis_scheme=AnalysisScheme.COLLOCATION, **kwargs):
 
         self.analysis_scheme = analysis_scheme
 
+        self.regular_phases = []
```

Comment:
I wonder if "flight_phases" might be better name than "regular_phases".

"Look, Big Daddy! It's regular Daddy! -- Ralph Wiggum

Reply:
that's a fair point. as long as we're consistent I'm ok.

Reply:
flight_phases kind of implies that it excludes taxi and groundroll, so I think "regular", "nominal", or "primary_mission" would be better

Reply:
We don't want to exclude taxi / groundroll. So I'm thinking we switch this back to regular_phases.

Reply:
primary_mission seems like longer than needed.